spec/descriptions/createSyntheticTest.md

```diff
@@ -0,0 +1,40 @@
+This API endpoint creates a Synthetic Test.
+
+## A note on testFrequency parameter:
+The unit of the testFrequency parameter is minute. 
+
+## Sample script and payload: 
+- A sample script to create a simple Ping API test
+
+```json
```

Comment:
**nitpick:** This is unfortunately not a valid JSON snippet. 😉

Reply:
True, I include the curl command here. Removed "json"